_docs/addons/project/mediapool_delete_protection.md

```diff
@@ -0,0 +1,161 @@
+---
+title: Pagination
+authors: [dpf-dd, skerbis]
```

Comment:
```suggestion
authors: [dpf-dd]
```

Reply:
Bitte Korrektur durchführen. "Add suggestion to batch" @dpf-dd

Reply:
Boah überall Buttons und Links und Fenster hier bei GitHub - sorry da steig ich nicht durch.
Ich hatte auf "Add suggestion to batch" geklickt und hoffe das dies ausreichend war um den Konflikt zu lösen.

Andernfalls erbitte ich Hilfe in Form weiterer Instruktionen.
(Ich nutze GitHub ausschließlich, um Redaxo auch mal was zurückzugeben. Ansonsten ein Buch mit Sieben Siegeln...)

Vielen Dank!